text/0317-third-party-dependency-management.md

```diff
@@ -31,13 +32,21 @@ for the details).
 At build time of these packages,
 we generate an `npm-shrinkwrap.json` file with the exact version of each dependency
 (including transitive ones).
-This means a customer installing those packages in a given version will always get all of their dependencies in a specific version
+This means a customer installing those packages using `npm` in a given version will always get all of their dependencies in a specific version
 (the one defined in `npm-shrinkwrap.json`) as well.
 
+**Note**: the Yarn package manager ignores the `npm-shrinkwrap.json` file,
```

Comment:
Not sure that this is good enough... I don't think it will be possible to "blame yarn" in case of a major vulnerability.